x/ccv/provider/keeper/grpc_query.go

```diff
@@ -346,30 +345,41 @@ func (k Keeper) QueryConsumerValidators(goCtx context.Context, req *types.QueryC
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	for _, v := range consumerValSet {
+	for _, consVal := range consumerValSet {
```

Comment:
nit: rename `consVal` to `consumerVal` as we usually used cons for consensus (see `consAddr`).